src/modules/notifications/controllers/notifications.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller, Get, Param, Patch, Put, Query, Session, UseGuards } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { IPagination } from '@studiohyperdrive/pagination';
+import { addMonths } from 'date-fns';
+
+import { Notification, NotificationStatus } from '../types';
+
+import { NotificationsQueryDto } from '~modules/notifications/dto/notifications.dto';
+import { NotificationsService } from '~modules/notifications/services/notifications.service';
+import { SessionHelper } from '~shared/auth/session-helper';
+import { LoggedInGuard } from '~shared/guards/logged-in.guard';
+
+@UseGuards(LoggedInGuard)
+@ApiTags('Notifications')
+@Controller('notifications')
+export class NotificationsController {
+	constructor(private notificationsService: NotificationsService) {}
+
+	@Get()
+	public async getNotifications(
+		@Query() queryDto: NotificationsQueryDto,
+		@Session() session: Record<string, any>
+	): Promise<IPagination<Notification>> {
+		const userInfo = SessionHelper.getArchiefUserInfo(session);
+		const notifications = await this.notificationsService.findNotificationsByUser(
+			userInfo.id,
+			addMonths(new Date(), -1).toISOString(),
+			queryDto.page,
+			queryDto.size
+		);
+		return notifications;
+	}
+
+	@Patch(':notificationId')
```

Comment:
misschien `@Patch(':notificationId/read)` ? dan is het duidelijk wat er gebeurt; nu lijkt het dat je n body kan meesturen met wat er gepatched moet worden

Reply:
yep, blijkbaar waren mijn changes daarvoor gestashed en dus niet opgenomen in deze PR.

Nieuwe PR staat klaar:
https://github.com/viaacode/hetarchief-proxy/pull/68